x/stakeibc/keeper/reward_converter_test.go

```diff
@@ -344,3 +347,61 @@ func (s *KeeperTestSuite) TestSwapRewardTokens() {
 	err = s.App.StakeibcKeeper.SwapRewardTokens(s.Ctx, rewardAmount, route)
 	s.Require().ErrorContains(err, "epoch not found")
 }
+
+func (s *KeeperTestSuite) TestPoolPriceQuery() {
```

Comment:
So this is only the successful route, should we put "_Successful" on the end of the name?
What are the failure cases which would be worth checking?  
Non-legal connection id causes fail? Epoch tracker mismatch (would have caught the issue earlier with Hour vs Stride Epochs) Etc.

Reply:
I think there's only two error cases, epoch tracker and connection ID. I don't think it's possible to force the unmarshal to fail

I'm just going to add these to the bottom so we don't have to do make a new setup function

Reply:
done